scss/accessories.css

```diff
@@ -0,0 +1,95 @@
+.accesories-headline {
```

Comment:
You shouldn't compile every file separately, just have one CSS file with all the styles that you use in the HTML file.

Reply:
You're right, now I have it done that way